jit/src/main/java/org/dcsa/conformance/standards/jit/checks/JitChecks.java

```diff
@@ -0,0 +1,118 @@
+
+  static final Predicate<JsonNode> IS_PORT_CALL_SERVICE = node -> node.has(SPECIFICATION);
+
+  static JsonRebaseableContentCheck checkPortCallService(PortCallServiceType serviceType) {
+    return JsonAttribute.ifThen(
+        "Expected Port Call Service type should match scenario (%s).".formatted(serviceType.name()),
+        IS_PORT_CALL_SERVICE,
+        JsonAttribute.mustEqual(
+            "Check if the correct Port Call Service was supplied.",
+            JsonPointer.compile("/" + SPECIFICATION + "/portCallServiceType"),
+            serviceType::name));
+  }
+
+  static JsonContentCheck checkRightFieldValues() {
+    return JsonAttribute.customValidator(
+        "Check if valid combinations of values are supplied.",
+        body -> {
+          Set<String> issues = new HashSet<>();
+          if (IS_PORT_CALL_SERVICE.test(body)) {
+            String actualServiceType = body.get(SPECIFICATION).get("portCallServiceType").asText();
+            issues.add(verifyPortCallServiceEventTypeCode(body, actualServiceType));
+            issues.add(verifyPortCallPhaseTypeCode(body, actualServiceType));
+          }
+          return issues.stream().filter(Objects::nonNull).collect(Collectors.toSet());
+        });
+  }
+
+  private static String verifyPortCallPhaseTypeCode(JsonNode body, String actualServiceType) {
+    String portCallPhaseTypeCode = body.get(SPECIFICATION).path("portCallPhaseTypeCode").asText("");
+    if (!PortCallPhaseTypeCode.isValidCombination(
+            PortCallServiceType.fromName(actualServiceType), portCallPhaseTypeCode)) {
+      return "Expected matching Port Call Service type with PortCallPhaseTypeCode. Found non-matching type: '%s' combined with code: '%s'"
+          .formatted(actualServiceType, portCallPhaseTypeCode);
+    }
+    return null;
+  }
+
+  private static String verifyPortCallServiceEventTypeCode(
+      JsonNode body, String actualServiceType) {
+    String portCallServiceEventTypeCode =
+        body.get(SPECIFICATION).get("portCallServiceEventTypeCode").asText();
+    if (!PortCallServiceEventTypeCode.isValidCombination(
+        PortCallServiceType.fromName(actualServiceType), portCallServiceEventTypeCode)) {
+      return "Expected matching Port Call Service type with PortCallServiceEventTypeCode. Found non-matching type: '%s' combined with code: '%s'"
+          .formatted(actualServiceType, portCallServiceEventTypeCode);
+    }
+    return null;
+  }
+
+  public static ActionCheck createChecksForTimestamp(
+      UUID matchedExchangeUuid, String expectedApiVersion, DynamicScenarioParameters dsp) {
+    List<JsonContentCheck> checks = new ArrayList<>();
+    if (dsp != null && dsp.currentTimestamp().classifierCode().equals(JitClassifierCode.PLN)) {
+      checks.add(checkPlannedMatchesRequestedTimestamp(dsp));
+    }
+    if (checks.isEmpty()) return null;
+    return JsonAttribute.contentChecks(
+        x -> true, matchedExchangeUuid, HttpMessageType.REQUEST, expectedApiVersion, checks);
```

Comment:
Does this (`x -> true`) work as intended? My understanding was that you would get this check on both roles with one of them being `?` since they did not perform the request side.